packages/components/pie-modal/src/index.ts

```diff
@@ -343,8 +347,9 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
             class="c-modal"
             size="${size}"
             position="${position}"
-            ?hasBackButton=${hasBackButton}
             ?hasActions=${leadingAction || supportingAction}
+            ?hasBackButton=${hasBackButton}
+            ?hasStackedActions=${hasStackedActions}
             ?isDismissible=${isDismissible}
             ?isFooterPinned=${isFooterPinned}
             ?isFullWidthBelowMid=${isFullWidthBelowMid}
```

Comment:
I am starting to wonder if using attributes for these vs class modifiers makes our lives easier or harder as the number of attributes grows. I suppose we'd be adding bloat either way but I'm not sure which approach makes more sense

Reply:
I personally prefer attributes, but it's totally just a coin-flip preference.

I don't think there's any major pro's and cons to either approach (as far as I can see anyway) but interested to know if others have any hard reasons for one or the other.